translations/es/dns-prefetch-control/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: DNS Prefetch Control
+permalink: /docs/dns-prefetch-control/
+---
+In short: this middleware lets you disable browsers' DNS prefetching by setting the `X-DNS-Prefetch-Control` header.
```

Comment:
In English: this file was not translated. Could we translate it (or delete most of it and add a link to [the English docs][0])?

En español: este archivo no está traducido. Quiero que lo traduces (o elimines el contento y añades un elace al [version íngles][0]), por favor.

[0]: https://helmetjs.github.io/docs/dns-prefetch-control/